packages/eas-cli/src/commands/update/index.ts

```diff
@@ -247,8 +249,15 @@ export default class UpdatePublish extends EasCommand {
       }
     }
 
-    const runtimeVersions = await getRuntimeVersionObjectAsync(exp, platformFlag, projectDir);
-    await checkEASUpdateURLIsSetAsync(exp, projectId);
+    const [runtimeVersions, expUpdated] = await getRuntimeVersionObjectAsync(
```

Comment:
Prefer to keep this named `exp` and renaming the one before it `expBeforeRuntimeVersionUpdate` (or using `let` is fine as well). The reason for this is that L429 is missing a change to `expUpdated`.